static/js/reducers/index_test.js

```diff
@@ -22,8 +24,15 @@ describe('reducers', () => {
   });
 
   describe('posts reducer', () => {
+    let getPostStub;
     beforeEach(() => {
       dispatchThen = store.createDispatchThen(state => state.posts);
+      getPostStub = sandbox.stub(api, 'getPost');
+      getPostStub.callsFake(async (id) => {
+        let post = makePost();
+        post.id = id;
+        return Promise.resolve(post);
```

Comment:
you should be able to just `return post;` here, in a function declared with `async` the return value is automatically wrapped in `Promise.resolve`.